File: L0 first/midterm/ex3.py
def falsetrue():
    l=[]
    k=[]
    for i in range(0,8):
        x=int(input("Enter number:"))
        if x%2==0:
            l.append(x)
        else:
            k.append(x)
    if len(k)!=0 and len(l)==0:
        print(True)
    else:
        print(False)
falsetrue()

    

File: L0 second/PW1/ex3.c
#include<stdio.h> 
int main(){
    float x1,x2,d;  
    printf("Enter first number:");
    scanf("%f",&x1);
    printf("Enter second number:");
    scanf("%f",&x2); 
    //if we do not write the condition, the program can print negative number
    if (x1>x2){ // if x1 bigger than x2,program will find x1-x2
    d=x1-x2;
    }
    else { //else prigram will find x2-x1 
        d=x2-x1;
    }
    printf("The absolute value of the difference between %f and %f is %f\n",x1,x2,d);
    return 0;
}